Makefile

```diff
@@ -1,6 +1,8 @@
 .DEFAULT_GOAL := help
 
-.PHONY: vm
+.PHONY: vm rustdeps juno juno-cached check-rust generate clean-testcache test test-cached \
```

Comment:
in my opinion we need it only for `vm` and `juno` + also consider to move them on top of corresponding commands